src/test/java/org/apache/sling/jcr/repoinit/GeneralAclTest.java

```diff
@@ -270,9 +270,9 @@ private void verifyRegisterNamespace(String username, String prefix, String uri,
         try {
             userSession = U.loginService(username);
             userSession.getWorkspace().getNamespaceRegistry().registerNamespace(prefix, uri);
-            assertTrue("Register namespace succeeded " + prefix + uri, successExpected);
+            assertTrue("Register namespace succeeded " + prefix + " " + uri, successExpected);
         } catch (RepositoryException e) {
-            assertTrue("Error registering namespace " + prefix + uri + " " + e.getMessage(), !successExpected);
+            assertFalse("Error registering namespace " + prefix + " " + uri + " " + e.getMessage(), successExpected);
```

Comment:
???? what does the changes for SLING-12115 lead to a changed assertion in a namespace related test? that doesn't look good at all.

Reply:
True, that's cleanup.

Reply:
but unrelated to the task at hand, right?
i really would like changes to a particular bug or improvement to be limited to the task at hand.... for me just looking at the diff it's just harder to figure out why something was change.